Sources/MockoloFramework/Utils/StringExtensions.swift

```diff
@@ -251,3 +242,174 @@ extension StringProtocol {
         return self.dropFirst(String.importSpace.count).trimmingCharacters(in: .whitespaces)
     }
 }
+
+extension TokenKind {
+    init?(keyword: String) {
```

Comment:
This initialize is very fat, but this will be implemented in future release of SwiftSyntax. https://github.com/apple/swift-syntax/blob/main/Sources/SwiftSyntax/gyb_generated/TokenKind.swift#L146